README.md

```diff
@@ -1,17 +1,65 @@
-go-libipfs
-=======================
+<h1 align="center">
+go-libipfs 🍌
+<br>
+<img src="https://raw.githubusercontent.com/ipfs/go-libipfs/feat/readme/logo.svg" alt="go-libipfs logo" title="go-libipfs logo" width="200">
+<br>
+</h1>
+<p align="center" style="font-size: 1.2rem;">A library for building IPFS applications and implementations.</p>
 
-> A library for building IPFS implementations
+<hr />
 
-Go-libips is a library for building IPFS implementations and tools in Go. It contains reusable functionality useful for interacting and experimenting with IPFS.
+[![Go Test](https://github.com/ipfs/go-libipfs/actions/workflows/go-test.yml/badge.svg)](https://github.com/ipfs/go-libipfs/actions/workflows/go-test.yml)
+[![Go Docs](https://img.shields.io/badge/godoc-reference-blue.svg)](https://pkg.go.dev/github.com/ipfs/go-libipfs)
+[![codecov](https://codecov.io/gh/ipfs/go-libipfs/branch/main/graph/badge.svg?token=9eG7d8fbCB)](https://codecov.io/gh/ipfs/go-libipfs)
```

Comment:
Sweet.  Do we have any CI checks to make sure we don't regress (at least that we don't regress when making non-import code changes)?

Reply:
Yes I will add them in a subsequent PR